source/electrical-power-systems/index.md

```diff
@@ -0,0 +1,46 @@
+# Electrical Power Systems
+
+Electrical trains need to be powered.
+In almost all cases, the best way to do this is not by having trains carry their own energy supply, as diesel and coal trains do, but to be connected to the electrical grid to receive externally-generated power continuously.
+Such power systems can be broken down into overhead and "third rail" systems.
+
+## Overhead Catenary Wire
+
+Catenary wire is strung up over the tracks.
+
+Rolling stock which needs power is connected to the overhead wire with a *pantograph*.
+
+Modern overhead wire systems for mainline rail are typically at higher voltages AC, such as 25 kV.
+
+## Third Rail
+
+A "third rail" was originally just that, a third rail not used to support the train, but instead power it.
+(Modern third rail has diverged in material and shape from regular rail, but the name has stuck.)
+
+The third rail is much thicker than a regular wire, and as such can carry higher currents.
+At the same time, being right at ground level, and less well insulated, the third rail requires lower voltage.
+Lower voltage but higher current roughly cancels out in terms of power (their product), however.
+
+### Inferiority
+
+The physical power delivery mechanism itself are much more expensive for third rail.
+Third rails only last about 20 years.[^third-rail-bart]
+Overhead wire itself and insulators, by contrast, typically lasts over twice as long as that[^overhead-wire-longevity],
+and the support poles often last even longer at over 100 years.
+Together, these costs and lifespans make the amortized costs of overhead wire construction and maintenance far lower.
+
+[^third-rail-bart]: <https://www.bart.gov/news/articles/2019/news20190731>
+
+[^overhead-wire-longevity]: <https://www.sdtools.com/pdf/WCRR11_fatigue.pdf>
+
+Third rail also requires the amount of supporting infrastructure needed.
+Because a third rail can only handle low voltages, each substation can only feed a few miles of track.
```

Comment:
can we quantify this? because the actual numbers here might be more explanatory than just a few.

Reply:
I am not sure, do we have sources about this in general? https://mas.to/@25kV/110962472389210219 I am not sure how to incorporate as it is comparing different types of overhead wire methods.